The new dude (test)

mprices opened the pull request:
I pulled out the frames for a new dude walking and put it in for testing. He looks pretty cool. I will need to tweak some frame animations to make it smoother. And the level design would have to change to accomodate a bigger dude.

mprices committed:
The new dude (test)

I pulled out the frames for a new dude walking and put it in for testing. He looks pretty cool. I will need to tweak some frame animations to make it smoother. And the level design would have to change to accomodate a bigger dude.

nicolasbock reviewed (commented):
Nice! You could play around with [setScale()](https://photonstorm.github.io/phaser3-docs/Phaser.Physics.Arcade.Sprite.html#setScale__anchor). That should allow you to change the dude's size.

nicolasbock committed:
Scale the dude so he fits

Signed-off-by: Nicolas Bock <nicolasbock@gmail.com>

nicolasbock commented:
Check out the scale function I added. Now he fits into the maze again :smile:

mprices committed:
A new (ninja) dude...

Put in the walking, idle, and jumping animations. Not sure why the idle and jumping don't seem to work quite right... but the dude is cool!

nicolasbock committed:
Add raw sprite assets

Signed-off-by: Nicolas Bock <nicolasbock@gmail.com>

nicolasbock committed:
Add boy ninja idle sprite sheet

Signed-off-by: Nicolas Bock <nicolasbock@gmail.com>

nicolasbock committed:
Fix idle animation

Signed-off-by: Nicolas Bock <nicolasbock@gmail.com>

nicolasbock committed:
Fix dude run animation

Signed-off-by: Nicolas Bock <nicolasbock@gmail.com>

nicolasbock committed:
Fix jump animation

Signed-off-by: Nicolas Bock <nicolasbock@gmail.com>

nicolasbock commented:
I resized some of the sprites to make them appear a bit fuzzier like the stone tiles.
